apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -90,13 +90,15 @@ DropletAutocompleteParameterTooltipManager.prototype.onCursorMovement_ = functio
     return;
   }
 
+  // we check the command name to make sure that we aren't opening the param dropdown
+  // when users are navigating through the code with arrow keys.
+  // Otherwise the cursor could get stuck in the dropdown
```

Comment:
super nit: Capitalise the start of each sentence and add a period at end of each.  This makes it easier to read longer comments :)